oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/IntKeyLongValueHashMap.java

```diff
@@ -57,6 +58,9 @@ public String toStorageData() {
 
     @Override
     public void toObject(String data) {
+        if (Strings.isNullOrEmpty(data)) {
```

Comment:
Personally, I want to say, this is harmful change. This `data` should not generate if there is no value of this column. `Calculate` func should make sure the `value` column is generated rather than using this `null` check. Such as `PercentileMetrics#calculate`, it guarantees the 5 values generated permanently. If there is no value, put the default value based on this metrics func, otherwise, which value should be shown up in the final query?

Reply:
@hanahmily Any feedback?